icc-x-api/icc-patient-x-api.ts

```diff
@@ -1052,6 +1052,417 @@ export class IccPatientXApi extends IccPatientApi {
     })
   }
 
+  shareUsingPost(
```

Comment:
Except some methods, the content of the shareUsingPost is the same than the `share` method. 
Please reuse the common code and avoid duplication